docs/source/markdown/podman-images.1.md.in

```diff
@@ -78,6 +78,7 @@ Valid placeholders for the Go template are listed below:
 
 | **Placeholder** | **Description**                                            |
 |-----------------|------------------------------------------------------------|
+| .Arch           | Architecture of image (omitted if manifest list)           |
```

Comment:
mhh, I think this is wrong to some extend because these new fields are never populated via the cli only the API so this would be super misleading to users.
Given the completion automation has no idea how to skip fields and I think cli and API should behave the same I think it would make the most sense to just set ExtendedAttributes also when called from the cli.

Reply:
I'm completely lost here. Does this answer everybody's question?
```diff
diff --git a/hack/xref-helpmsgs-manpages b/hack/xref-helpmsgs-manpages
index 627aa2e29..7f6e49189 100755
--- a/hack/xref-helpmsgs-manpages
+++ b/hack/xref-helpmsgs-manpages
@@ -52,7 +52,7 @@ my $Format_Exceptions = <<'END_EXCEPTIONS';
 # Deep internal structs; pretty sure these are permanent exceptions
 events       .Details
 history      .ImageHistoryLayer
-images       .ImageSummary
+images       .Arch .ImageSummary .Os
 network-ls   .Network
 
 # FIXME: this one, maybe? But someone needs to write the text
```

Reply:
yes I think that should work but also needs `IsManifestList`